srcpkgs/lazarus/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'lazarus'
+pkgname=lazarus
+version=1.8.4
+revision=1
+wrksrc=lazarus
+build_style=gnu-makefile
+makedepends="fpc qt5pas qt5-x11extras-devel"
+depends="fpc gdb fpc-source qt5pas"
+short_desc="Cross-platform IDE for FreePascal"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="GPL-2.0-or-later, LGPL-2.0-or-later, modified LGPL-2.0-or-later"
+homepage="https://www.lazarus-ide.org/"
+distfiles="https://sourceforge.net/projects/${pkgname}/files/Lazarus%20Zip%20_%20GZip/Lazarus%20${version}/${pkgname}-${version}.tar.gz"
+checksum=5a9801ac341a322ad1fa104d373f154d8ab72ba0add1693d0dcc8d936e6c10e9
+nopie=yes
+nocross=yes
+#Dependencies only exist on x86_64, x86_64-musl, i686 and it does not build on x86_64-musl.
+only_for_archs="x86_64 i686"
+
+do_build() {
+	make bigide LCL_PLATFORM=qt5
```

Comment:
please add ${makejobs} if applicable

Reply:
I will check what happens with make jobs in all three relevant packages (qtpass, lazarus, lazarus-lcl-gtk2).